package.json

```diff
@@ -83,13 +83,13 @@
     "typescript-tslint-plugin": "^0.5.4"
   },
   "resolutions": {
+    "ganache": "npm:@soloseng/ganache@7.8.0-beta.1",
```

Comment:
is this the final package ? shouldn't we have it under celo organization ?

Reply:
I would really like to merge this now, I opened [a ticket](https://github.com/celo-org/celo-monorepo/issues/10692) to move this later, or we may not need it at all after ganache is gone.